worker/index.ts

```diff
@@ -100,6 +105,18 @@ async function processRequest(skinService: MojangRequestService, interpreted: Cr
         case RequestedKind.Skin: {
             return await skinService.retrieveSkin(interpreted, gatherer);
         }
+        case RequestedKind.Cape: {
+            const cape = await skinService.retrieveCape(interpreted, gatherer);
+            if (cape.status === 404) {
+                return new Response(JSON.stringify({ error: "User cape could not be found"}), {
```

Comment:
Now that I know this better, maybe it's better to return a 404 with a blank 1x1 image here. Would be interesting to see if the browser could handle this gracefully.

Reply:
Pushed, seems browsers have no problem rendering a single transparent pixel in my testing.